tests/ops/api/v1/endpoints/test_privacy_request_endpoints.py

```diff
@@ -2958,29 +2969,34 @@ def test_verify_identity_no_admin_approval_needed(
 
         assert mock_run_privacy_request.called
 
-        assert mock_dispatch_email.called
+        assert mock_dispatch_message.called
 
-        call_args = mock_dispatch_email.call_args[1]
+        call_args = mock_dispatch_message.call_args[1]
         task_kwargs = call_args["kwargs"]
-        assert task_kwargs["to_email"] == "test@example.com"
+        assert task_kwargs["to_identity"] == Identity(
+            phone_number="+1 234 567 8910", email="test@example.com"
```

Comment:
Wouldn't this number fail the validator since it has spaces? If so do we also need to consider that when taking in the `Identity`?

Reply:
right now, I'm stripping the white space out, before I call Twilio. Because formatting could be different for each service, I'm hesitant to adjust the intake validator for `Identity`. But if we're OK with the standard phone number format for identity adhering to that format, I can definitely update that. Probably in a separate PR to keep things focused here.

Reply:
This makes sense. You are right, each service could have different requirements. With that I think we can leave things as they are.